input/pagecontent/smart-web-messaging.md

```diff
@@ -388,15 +410,222 @@ targetWindow.postMessage({
       // additional details as needed
     }
   }
-}, targetOrigin);
+}, ehrOrigin);
+```
+
+The EHR then performs the requested operation, creating an object in an internal
+scratchpad and assigning it id `123` before finally responding with:
+
+```js
+// From EHR -> app
+appWindow.postMessage({
+  "messageId": "<some new uid>",
+  "responseToMessageId": "<corresponding request messageId>",
+  "payload": {
+    "status": "201 Created",
+    "location": "ServiceRequest/123"
+  }
+}, appOrigin);
+```
+
+##### `scratchpad.read`
+
+The `scratchpad.read` operation allows for selection of either a single resource
+from the scratchpad by requesting its `location` handle, or for selection of the
+*entire* contents of the scratchpad by omitting `location` from the operation
+parameter list.
+
+###### Request `payload`
+
+| Property              | Optionality | Type   | Description |
+| --------------------- | ----------- | ------ | ----------- |
+| `location`            | OPTIONAL    | string | Takes the form `ResourceType/Id` when provided.  |
+{:.grid}
+
+###### Response `payload`
+
+| Property              | Optionality | Type   | Description |
+| --------------------- | ----------- | ------ | ----------- |
+| `resource`            | OPTIONAL    | object | A resource matching the requested `location`. |
```

Comment:
You say this right below the table, but it's worth including notes in the description, so folks see what's up.
* resource: "Returned when a scratchpad.read request asks to read a single resource on the scratchpad"
* resources: "Returned when a scratchpad.read request asks to read all resources on the scratchpad"

Reply:
See [this commit](https://github.com/HL7/smart-web-messaging/commit/918ea9b6b1096b783feeac41b8e12e246d639b51)